src/server.js

```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+import next from 'next';
+import nextI18NextMiddleware from 'next-i18next/middleware';
+
+import nextI18next from './i18n.ts';
+
+const port = process.env.PORT || 3000;
```

Comment:
Could be nice to default to another port than 3000 just because we have several projects running on 3000..

Reply:
But it might not matter since I think another port will be autoselected